src/ToDo.UI/App.xaml.host.cs

```diff
@@ -1,3 +1,7 @@
+#if DEBUG
+#define USE_MOCKS
+#endif
+
```

Comment:
This is not required anymore: it's not defined in the csproj.

Reply:
This should be left in for the moment - it's really annoying to work on the application without having mocks enabled, so they should be no by default in debug imho. Once my other PR goes through to push mocks to the config, then we can specify it in the appsettings.development.json which would be the best way